packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -2249,6 +2255,16 @@ export default class Meeting extends StatelessWebexPlugin {
     if (localStream) {
       const {audioTrack, videoTrack} = MeetingUtil.getTrack(localStream);
 
+      audioTrack.onmute = (e) => this.handleAudioLogging(e);
+      audioTrack.onunmute = (e) => this.handleAudioLogging(e);
+      videoTrack.onended = (e) => this.handleVideoLogging(e);
+      videoTrack.onmute = (e) => this.handleVideoLogging(e);
+
+      // TODO - did this trigger? any standard?
+      // audioTrack.addEventListener('unmute', (e) => this.handleAudioLogging(e));
```

Comment:
why do we have a todo here ?

Reply:
Piggybacking, let's remove any commented out code